host_core/lib/host_core/wasmcloud/runtime_server.ex

```diff
@@ -122,13 +125,24 @@ defmodule HostCore.WasmCloud.Runtime.Server do
 
   @impl true
   def handle_info(
-        {:invoke_callback, claims, binding, namespace, operation, payload, token},
+        {:invoke_callback, claims, {binding, namespace, operation}, payload, call_context, token},
         {_runtime, config} = state
       ) do
     Task.Supervisor.start_child(RuntimeCallSupervisor, fn ->
       # This callback is invoked by the wasmcloud::Runtime's host call handler
       payload = payload |> IO.iodata_to_binary()
 
+      with true <- is_list(call_context),
+           ctx_list <- :erlang.list_to_binary(call_context),
```

Comment:
this with statement is already a bit difficult to understand. if `ctx_list` is actually a binary, should we maybe call it `ctx_binary` instead?

Reply:
Sure 👍🏻